### DISCUSSION TITLE
Sean Boult /uses

### DISCUSSION THREAD

**Discussion by Hacksore** [Hacksore/Hacksore#26] [General]
My stuff

**Comment by star-ot** [comment]
what a nerd

**Reply by Hacksore** [reply]
![image](https://media.tenor.com/tla5JesXwOgAAAAC/cat-unimpressed.gif)

**Comment by PickleNik** [comment]
wtf is wezterm

**Reply by Hacksore** [reply]
not sure you might wanna google it 🤓
